Formula/hurl.rb

```diff
@@ -0,0 +1,39 @@
+class Hurl < Formula
+  desc "Run and Test HTTP Requests with plain text and curl"
+  homepage "https://hurl.dev"
+  url "https://github.com/Orange-OpenSource/hurl/archive/refs/tags/1.5.0.tar.gz"
+  sha256 "469cca44022d9339fad246fd417b22703258164fd1cdbe040476a1c127791184"
+  license "Apache-2.0"
+  head "https://github.com/Orange-OpenSource/hurl.git", branch: "master"
+
+  depends_on "python@3.9" => :build
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "curl"
+    depends_on "libxml2"
```

Comment:
Shouldn't these be `uses_from_macos`?

Reply:
Yes, much better! I've pushed the fixed lines